platform/android/src/jni_native.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include "jni.hpp"
+
+namespace mbgl {
+namespace android {
+
+extern void registerNatives(JavaVM* vm);
```

Comment:
is `extern` required here? functions are extern by default.

Reply:
It's not needed. Fixed!